syntax="proto3";
package pms;

option optimize_for = LITE_RUNTIME;

// the tag of storeage msg
enum EStorageTag{
    TSEQN           = 0;
    TDATA           = 1;
    TFETCHSEQN      = 2;
}

enum EStoreContType{
    TRAWCONT        = 0;
    TPACKEDSTORE    = 1;
}
